### PR TITLE
Setup test with initial test cases

### PR DESCRIPTION
**Pull request by kimquy**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -12,10 +12,12 @@ cache:
 
 install:
   - gem install rubocop
+  - gem install rake
```
Comment (gauravtiwari) [inline .travis.yml]: You are running `bundle exec rake` and rake isn't part of the Gemfile. Perhaps, add both rake and rubocop as dependencies of the Gem.

Comment (gauravtiwari) [inline .travis.yml]:
```
gem "rake", ">= 11.1"
gem "rubocop", ">= 0.47", require: false
```

**Comment by kimquy** [comment]
Thx @gauravtiwari for the feedback. Sorry that this PR take more commits than I expected.

File: Gemfile [hunk 2]
```diff
@@ -1,3 +1,8 @@
 source "https://rubygems.org"
 
 gemspec
+
+gem "rails"
+gem "rake", ">= 11.1"
+gem "minitest", "~> 5.0"
```
Comment (gauravtiwari) [inline Gemfile]: Lets move this to `:test` block - 

```rb
group :test do
  gem "minitest", "< 5.3.4"
end
```

**Comment by kimquy** [comment]
@gauravtiwari I updated the PR.

**Comment by gauravtiwari** [comment]
Looks good to me @kimquy 👍  🍰 

@rafaelfranca Please take a look when you have a chance.

File: test/env_test.rb [hunk 3]
```diff
@@ -0,0 +1,16 @@
+require "webpacker_test"
+
+class EnvTest < Minitest::Test
+  def test_current_env
+    assert Webpacker::Env.current == "production"
```
Comment (rafaelfranca) [inline test/env_test.rb]: `assert_equal`

File: test/env_test.rb [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+  end
+
+  def test_env_is_development?
+    refute Webpacker::Env.development?
```
Comment (rafaelfranca) [inline test/env_test.rb]: `refute_predicate`

File: test/env_test.rb [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+  end
+
+  def test_file_path
+    correct_path = File.join(File.dirname(__FILE__), "config", "webpack", "paths.yml").to_s
+    assert Webpacker::Env.file_path.to_s == correct_path
```
Comment (rafaelfranca) [inline test/env_test.rb]: `assert_equal`

File: test/webpacker_test.rb [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+require "rails/test_help"
+require "webpacker"
+
+module TestApp
+  class Application < ::Rails::Application
+    config.root = File.dirname(__FILE__)
+  end
+end
+
+TestApp::Application.initialize!
```
Comment (rafaelfranca) [inline test/webpacker_test.rb]: Do you need an application?

Comment (kimquy) [inline test/webpacker_test.rb]: @rafaelfranca Yes. it needs the `Application`!

**Pull request merged**